site/guide/work-with-content-blocks.qmd

```diff
@@ -50,6 +50,19 @@ Test-driven block
 After you have completed these steps, the new content block becomes a part of your model documentation.
 
 <!--- TO DO We might need to revisit terminology: we talk about "block", "section", and "module", but they all seem to be names for very similar things.--->
+### Add mathematical formulas
+
+While editing a content block, you can insert math equations using the editor. 
+
+#### Steps
```

Comment:
Since the parent is already at the H3 level, you could just omit this steps heading and end line 55 in a `:`.